src/main/java/com/sivalabs/ft/features/domain/ReportingService.java

```diff
@@ -0,0 +1,294 @@
+                        CSVWriter.RFC4180_LINE_END)) {
+
+            // Write headers
+            csvWriter.writeNext(CSV_HEADERS);
+
+            // Write data rows
+            for (RoadmapItemDto item : roadmapData.roadmapItems()) {
+                writeRoadmapItemToCsv(csvWriter, item);
+            }
+
+            return stringWriter.toString().getBytes();
+
+        } catch (IOException e) {
+            log.error("Error generating CSV export", e);
+            throw new RuntimeException("Failed to generate CSV export", e);
+        }
+    }
+
+    public byte[] exportRoadmapToPdf(RoadmapResponseDto roadmapData) {
+        log.debug(
+                "Exporting roadmap to PDF with {} items",
+                roadmapData.roadmapItems().size());
+
+        try {
+            ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+            Document document = new Document(PageSize.A4);
+            PdfWriter writer = PdfWriter.getInstance(document, outputStream);
+
+            document.open();
+
+            // Add title
+            addPdfTitle(document, roadmapData);
+
+            // Add summary section
+            addPdfSummary(document, roadmapData.summary());
+
+            // Add roadmap items
+            addPdfRoadmapItems(document, roadmapData.roadmapItems());
+
+            document.close();
+
+            return outputStream.toByteArray();
+
+        } catch (DocumentException e) {
+            log.error("Error generating PDF export", e);
+            throw new RuntimeException("Failed to generate PDF export", e);
+        }
+    }
+
+    public String generateExportFilename(String format) {
+        String timestamp = LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyyMMddHHmmss"));
+        return String.format("Roadmap_%s.%s", timestamp, format.toLowerCase());
+    }
+
+    private void writeRoadmapItemToCsv(CSVWriter csvWriter, RoadmapItemDto item) {
+        ReleaseDto release = item.release();
+        ProgressMetricsDto progress = item.progressMetrics();
+        HealthIndicatorsDto health = item.healthIndicators();
+
+        if (item.features().isEmpty()) {
+            // Write release info without features
+            String[] row = createCsvRow(release, progress, health, null);
+            csvWriter.writeNext(row);
+        } else {
+            // Write one row per feature
+            for (FeatureDto feature : item.features()) {
+                String[] row = createCsvRow(release, progress, health, feature);
+                csvWriter.writeNext(row);
+            }
+        }
+    }
+
+    private String[] createCsvRow(
+            ReleaseDto release, ProgressMetricsDto progress, HealthIndicatorsDto health, FeatureDto feature) {
+        List<String> row = new ArrayList<>();
+
+        // Release information
+        row.add(release.code().split("-")[0]); // Product code from release code
+        row.add(""); // Product name - would need to be passed separately
```

Comment:
What's the point of this? Why did you add `Product Name` column, if it's always empty?